indico/modules/events/registration/client/js/form/fields/NumberInput.jsx

```diff
@@ -10,7 +10,7 @@ import React from 'react';
 import {useSelector} from 'react-redux';
 import {Form, Label} from 'semantic-ui-react';
 
-import {FinalInput, FinalField, validators as v} from 'indico/react/forms';
+import {FinalInput, FinalField, validators as v, parsers} from 'indico/react/forms';
```

Comment:
I'd use `parsers as p` like we do for validators (just be consistent everywhere where we use it)